.github/workflows/cron.yml

```diff
@@ -6,15 +6,6 @@ on:
     - cron:  '0 5 * * *'
   # allow this to be scheduled manually in addition to cron
   workflow_dispatch:
-  push:
-    branches: [ main ]
-    paths:
-      - "tutorials/**"
-  pull_request:
-    branches: [ main ]
-    paths:
-      - "tutorials/**"
```

Comment:
weird, was the whole cron job triggered by every PR before?

Reply:
Just the PRs that change tutorials